tests/IceRpc.Tests/Transports/Slic/SlicTransportSslAuthenticationTests.cs

```diff
@@ -102,42 +84,34 @@ public async Task Ssl_server_connection_connect_fails_when_client_provides_untru
         var sut = provider.GetRequiredService<ClientServerMultiplexedConnection>();
         var listener = provider.GetRequiredService<IListener<IMultiplexedConnection>>();
 
-        // Start the TLS handshake by calling connect on the client and server connections and wait for the
-        // connection establishment.
+        // Start the TLS handshake.
         var clientConnectTask = sut.Client.ConnectAsync(default);
 
         // Act/Assert
         IMultiplexedConnection? serverConnection = null;
         Assert.That(
             async () =>
             {
-                // We accept two behaviors here:
-                // - the listener can internally kill the client connection if it's not valid (e.g.: Quic behavior)
-                // - the listener can return the connection but ConnectAsync fails (e.g.: Slic behavior)
-                using var cts = new CancellationTokenSource(TimeSpan.FromMilliseconds(250));
-                (serverConnection, _) = await listener.AcceptAsync(cts.Token);
+                (serverConnection, _) = await listener.AcceptAsync(default);
```

Comment:
Based on the comments on the original version, I would move this AcceptAsync outside the Assert.